CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
```

Comment:
According to https://keepachangelog.com/en/1.0.0/ you should use `Removed` here.

Reply:
I think I'll add a kind of a template, WDYT?